docs/_Docs_GetStarted.md

```diff
@@ -68,78 +73,79 @@ The final step for installation is copying over some of the files from our submo
 yarn run provision:token:contracts
 ```
 
-### Testing
+## `glider-rc.1` on the Görli testnet
 
-To deploy all contracts and run all tests, run the following command.
+The [Glider release candidate](/colonynetwork/docs-releases/) is in many ways a simpler and easier way to experiment than setting up a local development environment, and can be very useful if you're looking to just get a sense of how the colonyNetwork contracts work, or want to build extensions/integrations that remain inside the EVM.
 
-```
-yarn run test:contracts
-```
+To connect, you'll need to know the address of the colonyNetwork (which is, in reality, the address of the `etherRouter` contract; see [The Delegate Proxy Pattern](/colonynetwork/docs-the-delegate-proxy-pattern/) for more info).
 
-### Development
+`ColonyNetwork`: `0x79073fc2117dD054FCEdaCad1E7018C9CbE3ec0B`
 
-Alternatively, you can start a local test node and deploy the contracts yourself using the locally installed `truffle` package.
+You will also require Görli test ETH, and a deployed ERC20 token to import.
 
-```
-yarn run start:blockchain:client
 
-./node_modules/.bin/truffle migrate --reset --compile-all
-```
-
-For more detailed instructions, and additional steps required to set up an environment for use with [colonyJS](https://github.com/JoinColony/colonyJS), check out the colonyJS [Local Setup](/colonyjs/intro-local-setup/) documentation.
+### Access with Remix (good for experimenting)
 
-## Reputation Miner
+For simple interactions, [Remix](http://remix-alpha.ethereum.org/) is a good lightweight way to call specific functions and get network information from the contracts.
 
-The reputation mining client is usable for testing but it has limited functionality. It currently has no support for the challenge-response process to accommodate multiple submitted reputation root hashes. Still, it is possible to run a single miner instance for usable reputation scores on a local test network.
-
-### Start Mining Client
-
-You can start the reputation mining client using the following command.
+Rather than import the entire set of contracts into remix, use the included `solidity-steamroller` to flatten the needed interface contracts to the `build/flattened/` directory:
 
 ```
-node packages/reputation-miner/bin/index.js --colonyNetworkAddress { COLONYNETWORK_ADDRESS } --minerAddress { MINER_ADDRESS }
+$ yarn flatten:contracts
```

Comment:
You are missing `run` from this command

Reply:
As @area rightly pointed out to me earlier, this will work even without `run`. I didn't know that was the case but still I would probably want to be consistent and reference it still.